manifests/params_manager.pp

```diff
@@ -178,14 +178,13 @@
       $vulnerability_detector_provider_debian                 = 'yes'
       $vulnerability_detector_provider_debian_enabled         = 'no'
       $vulnerability_detector_provider_debian_os              = ['wheezy',
-        'stretch',
         'jessie',
```

Comment:
For `debian`, the correct values are `buster` and `bullseye`.

Reply:
done